csrankings-k.csv

```diff
@@ -695,6 +695,7 @@ Kin-hong Wong,Chinese University of Hong Kong,http://www.cse.cuhk.edu.hk/~khwong
 Kiona Ogle,Northern Arizona University,https://nau.edu/siccs/faculty/kiona-ogle,iNqaSrgAAAAJ
 Kiran Balagani,New York Tech,https://site.nyit.edu/bio/kbalagan,bU6NdUYAAAAJ
 Kiriakos N. Kutulakos,University of Toronto,http://www.cs.toronto.edu/~kyros,NOSCHOLARPAGE
+Kiril Solovey,Technion,https://kirilsol.github.io,b7sIEHcAAAAJ

```

Comment:
Not in CS.

Reply:
Kiril Solovey is a researcher in robotics, close in area to Oren Salzman, who is faculty at the CS department. The two have 18 publications together according to DBLP. All his degrees are in CS, and he was Oren's academic brother (they had the same PhD advisor, Dan Halperin). Kiril publishes in CS conferences such as SOCS and AAMAS.